bluetooth/src/commonMain/kotlin/device/DeviceConnectionManager.kt

```diff
@@ -53,8 +56,29 @@ abstract class BaseDeviceConnectionManager(
         set(value) { _currentAction.set(value) }
     protected val notifyingCharacteristics = sharedMutableMapOf<String, Characteristic>()
 
-    private val _mtu = AtomicInt(-1)
-    val mtu get() = _mtu.value
+    private val _rssiUpdate = createSharedFlow<Int>()
+    val rssiUpdate: Flow<Int> = _rssiUpdate
+    private val _startedConnecting = createSharedFlow<Unit>()
+    val startedConnecting: Flow<Unit> = _startedConnecting
+    private val _cancelledConnecting = createSharedFlow<Unit>()
```

Comment:
Multiple `Unit` state flows look. a little bit awkward for me. Can we replace it with an enum or a sealed class?

As long as it is tested and works it is not critical for me, but probably it makes sense ti create a separate ticket for refactoring?